src/pk_compiler.c

```diff
@@ -1078,7 +1078,7 @@ GrammarRule rules[] = {  // Prefix       Infix             Infix Precedence
   /* TK_WHILE      */   NO_RULE,
   /* TK_FOR        */   NO_RULE,
   /* TK_IF         */   NO_RULE,
-  /* TK_ELIF       */   NO_RULE,
+  /* TK_ELSIF     */   NO_RULE,
```

Comment:
Here the alignment is off by a white space, but that's okay, I'll merge it and fix it by myself. But it's in discussion at #74, I'd still like to hear from @xSavitar and @alexcpatel about the change and merge it by tomorrow.

Thanks!

Reply:
Thanks for the review. Good luck!